packages/connect/src/api/bitcoin/Fees.ts

```diff
@@ -21,6 +22,30 @@ const fillGap = (from: number, step: number, size: number) => {
     return fill;
 };
 
+const findNearest = (requested: number, blocks: Blocks) => {
```

Comment:
I've tried to simplify `findNearest` a bit, but it's probably less effective.
```
const findNearest = (requested: number, blocks: Blocks) => {
    const i = Math.max(requested, 0);
    return blocks
        .slice(i)
        .concat(blocks.slice(0, i).reverse())
        .find(item => typeof item === 'string');
};
```
But during that, I've found out that `requested` could be negative, specifically -1. Was it considered when implementing `findNearest`? It seems that the result is ok (same as if it was 0), I just want to make sure.

Reply:
`-1` means that we dont know what does requested fee level mean in term of "blocks", this happens for coins with only one defined fee level (normal) - so basically all of them [except for coins specified here
](https://github.com/trezor/trezor-suite/blob/develop/packages/connect/src/data/defaultFeeLevels.ts#L6)

in case of -1 we are trying to find first value by increasing requested index and walk forward until we find something - usually found on second increase (1)

Reply:
an the answer for question "Was it considered when implementing findNearest?" is no, because previously this case was using [findLowest](https://github.com/trezor/trezor-suite/blob/develop/packages/connect/src/api/bitcoin/Fees.ts#L135) util